### PR TITLE
chore: mode mainnet

### PR DESCRIPTION
**Pull request by molendtoby**

### CODE REVIEW

**Review by asimof — COMMENTED** [review]
Should we keep both `mode_sepolia` and `mode` networks in config?

File: hardhat.config.ts [hunk 1]
```diff
@@ -108,7 +108,8 @@ const buidlerConfig: HardhatUserConfig = {
     matic: getCommonNetworkConfig(ePolygonNetwork.matic, 137),
     mumbai: getCommonNetworkConfig(ePolygonNetwork.mumbai, 80001),
     // mode: getCommonNetworkConfig(eModeNetwork.mode, 31337),
-    mode: getCommonNetworkConfig(eModeNetwork.mode, 919),
+    mode: getCommonNetworkConfig(eModeNetwork.mode, 34443),
+    modeTestnet: getCommonNetworkConfig(eModeNetwork.modeTestnet, 919),
```
Comment (asimof) [inline hardhat.config.ts]: Should we use `modeSepolia`? It's possible they set up a new testnet after Sepolia sunsets.

File: markets/modeTestnet/index.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  ProviderId: 3,
+  ReservesConfig: {
+    USDC: strategyUSDC,
+    USDT: strategyUSDT,
+    WETH: strategyWETH,
+    WSTETH: strategyWSTETH,
+  },
+  // TODO mode
+  ReserveAssets: {
+    [eModeNetwork.mode]: {
```
Comment (asimof) [inline markets/modeTestnet/index.ts]: Should the network be mode testnet?

File: markets/modeTestnet/commons.ts [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+  LendingPool: {
+    [eModeNetwork.mode]: '',
+  },
+  LendingPoolConfigurator: {
+    [eModeNetwork.mode]: '',
+  },
+  // TODO mode
+  ProviderRegistry: {
+    // [ePolygonNetwork.mumbai]: '0xE6ef11C967898F9525D550014FDEdCFAB63536B5',
+    [eModeNetwork.mode]: ''
```
Comment (asimof) [inline markets/modeTestnet/commons.ts]: Should the network be mode testnet?

File: helpers/constants.ts [hunk 4]
```diff
@@ -85,7 +85,7 @@ export const chainlinkAggregatorProxy = {
   fuji: '0x5498BB86BC934c8D34FDA08E81D444153d0D06aD',
   tenderly: '0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419',
   // TODO ETH oracle
-  mode: '0x0201e9937DcCa9dC61315C2f44C2966cC0a33054',
+  mode: '0x5290b68e09b4f79F63ed01710003E7949FB32B77',
```
Comment (asimof) [inline helpers/constants.ts:87]: Should we also add contract for mode testnet?

**Review by asimof — CHANGES_REQUESTED** [review]

File: helpers/constants.ts (hunk 4 above)
Comment (molendtoby) [reply helpers/constants.ts:87]: it's down below this line

**Review by asimof — APPROVED** [review]

**Pull request merged**